.azure/tutorials-linux.yml

```diff
@@ -22,8 +22,8 @@ jobs:
         inputs:
           key: 'pip | "$(Agent.OS)" | "${{ parameters.pythonVersion }}"'
           restoreKeys: |
-            pip | "$(Agent.OS)"
-            pip
+            pip-2 | "$(Agent.OS)"
+            pip-2
```

Comment:
Did you want to strip this one too?

Reply:
oops I missed the file, let me drop this too